test/fixtures/multi-go-modules.yml

```diff
@@ -0,0 +1,46 @@
+# syntax=ghcr.io/azure/dalec/frontend:latest
+
+name: dalec-test-multi-go-modules-build
```

Comment:
Can we migrate this into an integration test under `test/`?
There should already be one in there for gomods we can extend for multi-module.